WalletWasabi.Fluent/ViewModels/Navigation/RoutableViewModel.cs

```diff
@@ -52,9 +52,9 @@ protected virtual void OnNavigatedTo(bool inStack, CompositeDisposable disposabl
 		{
 		}
 
-		private void DoNavigateFrom()
+		private void DoNavigateFrom(bool isInHistory)
```

Comment:
inStack?

Reply:
in navigation history makes more sense when talking about the navigation from the viewmodels perspective... @soosr will do a seperate pr to make this consistent in codebase.